windows/ReactTestApp/Manifest.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <nlohmann/json.hpp>
+
+namespace winrt::ReactTestApp::implementation
+{
+    struct Component {
+        std::string appKey;
+        std::optional<std::string> displayName;
+        std::optional<std::map<std::string, std::string>> initialProperties;
+    };
+
+    struct Manifest {
+        std::string name;
+        std::string displayName;
+        std::vector<Component> components;
+    };
+
+    Manifest getManifest();
```

Comment:
Seems the naming convention in a UWP app is CamelCase, so we should use `GetManifest()` here.
```suggestion
    Manifest GetManifest();
```